ocean_lib/models/test/helpers.py

```diff
@@ -0,0 +1,18 @@
+from brownie.network import web3
+
+
+def interrogate_blockchain_for_reverts(
```

Comment:
no need for this anymore, I think

Reply:
I will remove `pytest.raises` block and  keep this function to retrieve the error message.